pkgs/by-name/mu/muvm/package.nix

```diff
@@ -9,44 +9,44 @@
   passt,
   sommelier,
   mesa,
+  systemd,
+  udev,
+  pkg-config,
   opengl-driver ? mesa.drivers,
   withSommelier ? false,
 }:
 
 rustPlatform.buildRustPackage rec {
-  pname = "krun";
-  version = "0-unstable-2024-06-18";
+  pname = "muvm";
+  version = "0.2.0";
 
   src = fetchFromGitHub {
-    owner = "slp";
+    owner = "AsahiLinux";
     repo = pname;
-    rev = "912afa5c6525b7c8f83dffd65ec4b1425b3f7521";
-    hash = "sha256-rDuxv3UakAemDnj4Nsbpqsykts2IcseuQmDwO24L+u8=";
+    rev = "muvm-${version}";
+    hash = "sha256-eB60LjI1Qr85MPtQh0Fb5ihzBahz95tXaozNe8q6o3o=";
   };
 
-  patches = [
-    (fetchpatch {
-      url = "https://github.com/slp/krun/pull/39.diff";
-      hash = "sha256-CV69L+VDDLRcWgpgDCAYKLlTU9ytFcHhzNgOibWD8KY=";
-    })
-    (fetchpatch {
-      url = "https://github.com/slp/krun/pull/38.diff";
-      hash = "sha256-cK3iDhh+33H16V65lWUXahjmpSxI1HhiLUmkjfkRB7A=";
-    })
-  ];
+  cargoHash = "sha256-ilCVALy1ofiq6Ak8nBhWusPwRAnD9VgOiOV5PKhF5GQ=";
 
-  cargoHash = "sha256-NahnigxJaY2QwWnySCRrnf3JyqZ+7jRA1CpE7ON0OOE=";
+  postPatch = ''
+    substituteAll crates/muvm/src/guest/bin/muvm-guest.rs \
+      --replace-fail "/usr/lib/systemd/systemd-udevd" "${lib.getExe' systemd "systemd-udevd"}"
```

Comment:
`systemd-udevd` is in `$out/lib`, not `$out/bin`, so `lib.getExe'` doesn't work here.

Reply:
Done